ui/components/ui/icon-button/icon-button.js

```diff
@@ -34,7 +40,11 @@ export default function IconButton(props) {
         <>
           <div
             data-theme="light"
-            className={classNames('icon-button__circle', iconButtonClassName)}
+            className={classnames(
+              'icon-button__circle',
+              'flex justify-center items-center h-9 w-9 bg-primary-default  rounded-full mt-1.5 mb-1 mx-auto',
+              iconButtonClassName,
+            )}
```

Comment:
These Tailwind classnames effectively replace the following CSS. We're retaining `icon-button__circle` for existing tests:

```css
  &__circle {
    display: flex;
    justify-content: center;
    align-items: center;
    height: 36px;
    width: 36px;
    background: var(--color-primary-default);
    border-radius: 99px;
    margin-top: 6px;
    margin-bottom: 4px;
    margin-inline: auto;
  }
```